src/qibocal/protocols/characterization/two_qubit_interaction/chevron.py

```diff
@@ -0,0 +1,296 @@
+        raise ValueError("You need to specify a list of pairs.")
+
+    # create a DataUnits object to store the results,
+    data = ChevronData()
+    for pair in qubits:
+        # order the qubits so that the low frequency one is the first
+        ord_pair = order_pairs(pair, platform.qubits)
+        # create a sequence
+        sequence = create_sequence(ord_pair, platform, params)
+        fx_pulse = sequence.qf_pulses[0]
+        ro_pulses = {pulse.qubit: pulse for pulse in sequence.ro_pulses}
+
+        # define the parameter to sweep and its range:
+        delta_amplitude_range = np.arange(
+            params.amplitude_min,
+            params.amplitude_max,
+            params.amplitude_step,
+        )
+        delta_duration_range = np.arange(
+            params.duration_min, params.duration_max, params.duration_step
+        )
+
+        sweeper_amplitude = Sweeper(
+            Parameter.amplitude,
+            delta_amplitude_range,
+            pulses=[fx_pulse],
+            type=SweeperType.ABSOLUTE,
+        )
+        sweeper_duration = Sweeper(
+            Parameter.duration,
+            delta_duration_range,
+            pulses=[fx_pulse],
+            type=SweeperType.ABSOLUTE,
+        )
+
+        results = platform.sweep(
+            sequence,
+            ExecutionParameters(
+                nshots=params.nshots,
+                acquisition_type=AcquisitionType.DISCRIMINATION,
+                averaging_mode=AveragingMode.CYCLIC,
+            ),
+            sweeper_amplitude,
+            sweeper_duration,
+        )
+        for qubit in ord_pair:
+            result = results[qubit]
+            prob = result.statistical_frequency
+            data.register_qubit(
+                ord_pair[0],
+                ord_pair[1],
+                qubit,
+                delta_duration_range,
+                delta_amplitude_range,
+                prob,
+            )
+
+    return data
+
+
+def _plot(data: ChevronData, fit: ChevronResults, qubits):
+    """Plot the experiment result for a single pair."""
+    fig = make_subplots(rows=1, cols=2, subplot_titles=("low", "high"))
+    states = ["low", "high"]
+    # Plot data
+    colouraxis = ["coloraxis", "coloraxis2"]
+
+    fit_report = ""
+    for state, qubit in zip(states, qubits):
+        index = (qubits[0], qubits[1], qubit)
+        ordered_index = index if index in data.data else (index[1], index[0], index[2])
+        fig.add_trace(
+            go.Heatmap(
+                x=data[ordered_index].length,
+                y=data[ordered_index].amp,
+                z=data[ordered_index].prob,
+                name=f"Qubit {qubit} |{state}>",
+                coloraxis=colouraxis[states.index(state)],
+            ),
+            row=1,
+            col=states.index(state) + 1,
+        )
+
+        fit_report += f"q{qubit} - {state} frequency| "
+        fit_report += f"Period of oscillation: {fit.period[str(ordered_index)]} ns<br>"
```

Comment:
```suggestion
       #  fit_report += f"Period of oscillation: {fit.period[str(ordered_index)]} ns<br>"
```
Comment it for now until we get the fitting